package.json

```diff
@@ -27,12 +27,12 @@
     "axios": "^1.6.4"
   },
   "devDependencies": {
-    "@types/node": "^14.18.12",
-    "prettier": "^2.2.1",
+    "@types/node": "^20.11.10",
+    "prettier": "^3.2.4",
     "typescript": "5.x"
   },
   "peerDependencies": {
-    "@types/node": "^14.14.22"
+    "@types/node": ">=14,<21"
```

Comment:
Was this change driven by any particular external force?

I'm curious since we've started seeing this seemingly related error from `npm audit fix` on repos depending on the `bankid` package:
```
npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name ">=14,<21" of package "@types/node@>=14,<21": Tags may not have any characters that encodeURIComponent encodes.
```

Reply:
Hey - good catch, there's a typo here. I've released a new version with that fixed :)

Reply:
Great, thanks!